codestyle/intellij-java-google-style.xml

```diff
@@ -0,0 +1,598 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<code_scheme name="GoogleStyle">
```

Comment:
Does anyone use these files? I do not think I use this in my Nessie project :thinking: IJ is generally pretty smart to figure out the code style from existing files and Nessie forces the stype via `spotless`.

I think we should just use `splotless` + google style from day 1 in this repo. WDYT?

Reply:
I just removed eclipse-related stuff and kept only IntelliJ stuff. I think IDE picks this file automatically if exists. 

I will enforce spotless once I have pom files (in the next PR).